toolkits/global/packages/global-expander/js/expander.js

```diff
@@ -152,18 +152,23 @@ const Expander = class {
 			this._triggerEl.dispatchEvent(event);
 		}
 
-		if (this._options.AUTOFOCUS) {
-			if (this._targetTabbableItems.length > 0) {
-				const firstTabbableItem = this._targetTabbableItems[0];
-				firstTabbableItem.focus();
+		switch (this._options.AUTOFOCUS) {
```

Comment:
I might be wrong, but this doesn't feel like it should be a switch statement to me, mainly because there's no default behaviour, it's just an if else if.

Further to that, should we be throwing an error if a string that we don't expect is passed in?

Reply:
I'll make new PR for this